src/resources/views/operation/modals/create_operation.blade.php

```diff
@@ -139,7 +139,14 @@
                             <select name="integration_id" id="create-operation-channel" style="width: 100%;">
                                 <option value=""></option>
                                 @foreach($notification_channels as $channel)
-                                <option value="{{ $channel->id }}">{{ $channel->name }}</option>
+                                @php
```

Comment:
instead injecting raw php, you can use blade syntax as bellow, making it more readable :

```
@if($channel->id == setting('kassie.calendar.slack_integration_default_channel', true))
<option value="{{ $channel->id }}" selected>{{ $channel->name }}</option>
@else
<option value="{{ $channel->id }}">{{ $channel->name }}</option>
@endif